### PR TITLE
Remove Visual Studio 2013 related files and documents

### PR DESCRIPTION
**Pull request by cosmo0920**
Currently, due to MariaDB, windows packaging batch files cannot build Mroonga Windows package.
We should remove them.

### CODE REVIEW

File: packages/windows/README.md [hunk 1]
```diff
@@ -6,16 +6,16 @@ TODO...
 
 ## Build with Visual C++ Express
 
-You need to use Visual C++ 2013 or later to build Mroonga with Express
-edition. `build-vc2013.bat` is a build batch script to build with
-Visual C++ Express 2013.
+You need to use Visual Studio 2015 for Windows Desktop or later to build Mroonga with express
+edition. `build-vc2015.bat` is a build batch script to build with
+Visual Visual Studio 2015 for Windows Desktop.
```
Comment (kenhys) [inline packages/windows/README.md]: too much Visual here?

Comment (cosmo0920) [inline packages/windows/README.md]: Oops! :confounded:

Comment (kenhys) [inline packages/windows/README.md]: I mean that **Visual Visual** is not correct.

Comment (cosmo0920) [inline packages/windows/README.md]: Removed duplicated _Visual_.

**Comment by kenhys** [comment]
Thanks!

**Pull request merged**